packages/renderer/src/lib/preferences/Util.spec.ts

```diff
@@ -37,13 +37,13 @@ test('write array object', () => {
 test('write invalid object', () => {
   writeToTerminal(xtermMock, {} as unknown as string[], 'test');
   // no error reported
-  expect(xtermMock.write).toBeCalledWith(expect.stringContaining('test'));
+  expect(xtermMock.write).not.toBeCalledWith(expect.stringContaining('test'));
```

Comment:
I don't see the rational here ?

Reply:
it is trying to write on the terminal an invalid value, so it should not call the `xterm.write` at all.
Now there is a check that only call `xterm.write` when it deals with a filled string array/string value.
No?